content/guides/composite_monitors.md

```diff
@@ -0,0 +1,161 @@
+
+#### One multi-alert monitor (monitor A)
+
+The composite monitor will have a multi-alert type. If monitor A has 4 reporting sources — hosts web01 through web04 — you may receive up to 4 alerts each time Datadog evaluates the composite monitor. In other words: for a given evaluation cycle, Datadog has 4 cases to consider. For each case, monitor A's status may vary across its sources, but the statuses of monitors B and C — which are simple alert types — are unchanging. Here's an example cycle:
+
+| monitor A | monitor B | monitor C | composite status |
+| --------- | --------- | --------- | ---------------- |
+| T (web01) | T         | T         | T - alert!       |
+| F (web02) | T         | T         | F                |
+| T (web03) | T         | T         | T - alert!       |
+| F (web04) | T         | T         | F                |
+{:.table}
+
+In this cycle, you would receive two alerts.
+
+#### Many multi-alert monitors (monitors A and B)
+
+The composite monitor will have a multi-alert type, but the number of alerts per cycle will be, at most, the number of common sources between monitors A and B. If web01 through web05 are reporting for monitor A, and web04 through web09 are reporting for monitor B, the composite monitor _only_ considers the common sources: web04 and web05. You can only receive up to two alerts in a cycle. Here's an example cycle:
+
+| monitor A | monitor B | monitor C | composite status |
+|-----------|-----------|-----------|------------------|
+| T (web04) | T (web04) | T         | T - alert!       |
+| F (web05) | T (web05) | T         | F                |
+{:.table}
+
+In this cycle, you would receive one alert.
+
+Remember: all multi-alert monitors used in a composite monitor _must_ use the same group-by. And if they do, and there are multiple groups in the group-by, the groups must appear in the same order. For example, a monitor grouped by `{host,environment}` and another grouped by `{environment,host}` cannot be used together in a composite monitor.
+
+### When to alert?
+
+The previous section considered monitor statuses in binary terms: true, or false. But a monitor can have statuses other than simply `Alert` and `OK`. Possible statuses include (in order of increasing severity): `OK`, `Skipped`, `Ignored`, `No Data`, `Unknown`, `Warn`, and `Alert`. Above, what true and false actually refer to is whether or not a given status should trigger an alert (true) or not (false). In other words: the status' **alert-worthiness**. 
+
+Composite monitors consider statuses from `No Data` up to `Alert` to be alert-worthy. When a composite monitor evaluates as alert-worthy, it inherits the most severe status among its individual monitors. When a composite monitor does not evaluate as alert-worthy, it inherits the _least_ severe status.
```

Comment:
It might also be worth saying explicitly that this means that an alert may happen even though they do not all have the same status.